package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "ekko-lightbox",
   "description": "A lightbox gallery plugin for Bootstrap based on the modal plugin",
-  "version": "5.2.0",
```

Comment:
I'll bump this to 5.3.0 before publishing to npm as there's some significant feature changes here